### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by ashwath22**
test @jaytolentino

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,5 @@ Simple web app for voters to make informed decisions.
 
 
 At Ballot Builder, we want to help citizens make informed decisions while choosing candidates for all positions.
+
+Adding more stuff in here
```
Comment (jaytolentino) [inline README.md]: Looks awesome! 😄 🐱 🌵

File: README.md [hunk 2]
```diff
@@ -3,3 +3,5 @@ Simple web app for voters to make informed decisions.
 
 
 At Ballot Builder, we want to help citizens make informed decisions while choosing candidates for all positions.
```
Comment (jaytolentino) [inline README.md]: Can you change the period to an exclamation point at the end? To show our enthusiasm! 😜

Comment (ashwath22) [reply README.md]: done!

**Comment by jaytolentino** [comment]
@ashwath22 great thanks for adding that!

**Pull request merged**